src/use-format-block.hook.js

```diff
@@ -6,10 +6,35 @@ export function useFormatBlock() {
   return {
     formatBlock(value) {
       if (value === document.queryCommandValue('formatBlock')) {
-        performCommandWithValue('div')
+        if (!!window.chrome && isInHTMLList() && document.queryCommandValue('formatBlock').includes('h')) {
+          removeHeader()
+        }
+        else {
+          performCommandWithValue('div')
+        }
       } else {
         performCommandWithValue(value)
       }
     },
   }
+}
+
+function isInHTMLList() {
+  return document.queryCommandState('insertOrderedList') || document.queryCommandState('insertUnorderedList')
+}
+
+function removeHeader() {
+  let nodeToBeRemoved = window.getSelection().getRangeAt(0).commonAncestorContainer
+  let index = 0
+  while (!['H1', 'H2', 'H3', 'H4', 'H5', 'H6'].includes(nodeToBeRemoved.tagName) && index < 5) {
+    nodeToBeRemoved = nodeToBeRemoved.parentNode
+    index ++
+  }
+  if (index === 4) { // no header found, contradicting line 9 qcv check, hopefully never happens
+    performCommandWithValue('div')
```

Comment:
should we `return` after doing this, instead of proceeding to remove nodes?

Reply:
and actually i've changed my mind on this -- I think your `throw Error()` was the correct thing to do in this situation. I didn't realize at first that this if statement should *never* be hit. Let's change it back 😄